docs/guides/events.md

```diff
@@ -0,0 +1,118 @@
+# Events
```

Comment:
Sounds like this is for EventTarget, so, we should rename the file and this title to be Event Target.